core/state_processor_test.go

```diff
@@ -354,3 +359,51 @@ func GenerateBadBlock(parent *types.Block, engine consensus.Engine, txs types.Tr
 	// Assemble and return the final block for sealing
 	return types.NewBlock(header, txs, nil, receipts, trie.NewStackTrie(nil), nil, true)
 }
+
+func TestStateProcessor_Process(t *testing.T) {
+	database := rawdb.NewMemoryDatabase()
```

Comment:
No need to defer closing it or smth like that?

Reply:
I defer closing the blockchain itself